app/components/decision/decision-result-pill.hbs

```diff
@@ -1,7 +1,22 @@
-<Auk::Pill
-  data-test-decision-result-pill={{true}}
-  @skin={{this.skin}}
-  @icon={{this.icon}}
->
-  {{@decisionResultCode.label}}
-</Auk::Pill>
\ No newline at end of file
+{{#if (not @isEditable)}}
+  <AuPill
+    @skin={{this.skin}}
+    @icon={{this.icon}}
+    data-test-decision-result-pill
+    data-test-agendaitem-decision
+  >
+    {{this.label}}
+  </AuPill>
+{{else}}
+  <AuPill
+    @skin={{this.skin}}
+    @icon={{this.icon}}
+    @onClickAction={{@onEdit}}
+    @actionIcon="pencil"
+    @actionText={{t "edit"}}
+    data-test-decision-result-pill
+    data-test-agendaitem-decision
+  >
```

Comment:
`data-test-agendaitem-decision` lijkt niet gebruikt en mag weg.
De dubbele selector is ok voor nu.

Reply:
zal sffs nog eens bekijken, ik had ergens gezocht en maar 1 ref gevonden maar mogelijk had ik filters aanstaan ofzo.

Reply:
Ik was idd niet helemaal juist.
Ik ga wel 1 van de selectors eruit halen en aanpassen waar nodig.